### PR TITLE
Add htsPreCompile get tokenInfo acceptance tests

### LINKED ISSUE #400: Add htsPreCompile get tokenInfo acceptance tests
### Problem

HIP 514 get tokenInfo methods do not have acceptance test support

### Solution

Add acceptance test methods for

- `getFungibleTokenInfo(address)`
- `getTokenInfo(address)`
- `getNonFungibleTokenInfo(address, int64)`

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by ar-conmit**
**Description**:
Add tests for:
* getFungibleTokenInfo
* getTokenInfo
* getNonFungibleTokenInfo - depends on `mintToken`
* mintToken

**Related issue(s)**:

Fixes #400 

**Notes for reviewer**:
The branch is created from `399-add-htsPrecompile-allowance-approval-tests`, some of the new logic was already implemented there.

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: packages/server/tests/acceptance/htsPrecompile.spec.ts [hunk 1]
```diff
@@ -30,64 +30,206 @@ import ERC20MockJson from '../contracts/ERC20Mock.json';
...
+    const baseHTSContract = new ethers.Contract(BaseHTSContractAddress, BaseHTSJson.abi, accounts[0].wallet);
+
+    const tx = await baseHTSContract.getFungibleTokenInfoPublic(HTSTokenContractAddress);
+
+    const { token, totalSupply } = (await tx.wait()).events.filter(e => e.event === 'FungibleTokenInfo')[0].args.tokenInfo.tokenInfo;
+
+    expect(totalSupply.toNumber()).to.equal(TOKEN_MAX_SUPPLY);
+    expect(token.maxSupply).to.equal(TOKEN_MAX_SUPPLY);
+    expect(token.name).to.equal(TOKEN_NAME);
+    expect(token.symbol).to.equal(TOKEN_SYMBOL);
```
Comment (Nana-EC) [inline packages/server/tests/acceptance/htsPrecompile.spec.ts:185]: Add coverage for decimal also

File: packages/server/tests/acceptance/htsPrecompile.spec.ts [hunk 2]
```diff
@@ -30,64 +30,206 @@ import ERC20MockJson from '../contracts/ERC20Mock.json';
...
+  it('should be able to get non-fungible token info', async () => {
+    const baseHTSContract = new ethers.Contract(BaseHTSContractAddress, BaseHTSJson.abi, accounts[0].wallet);
+
+    const tx = await baseHTSContract.getNonFungibleTokenInfoPublic(NftHTSTokenContractAddress, NftSerialNumber);
+
+    const { token, totalSupply } = (await tx.wait()).events.filter(e => e.event === 'NonFungibleTokenInfo')[0].args.tokenInfo.tokenInfo;
+
+    expect(totalSupply.toNumber()).to.equal(NftSerialNumber);
+    expect(token.name).to.equal(TOKEN_NAME);
+    expect(token.symbol).to.equal(TOKEN_SYMBOL);
```
Comment (Nana-EC) [inline packages/server/tests/acceptance/htsPrecompile.spec.ts:233]: Add coverage for serial number

**Review by Nana-EC — CHANGES_REQUESTED** [review]
LG, just some missing property coverage

File: packages/server/tests/acceptance/htsPrecompile.spec.ts (hunk 2 above)
Comment (ar-conmit) [reply packages/server/tests/acceptance/htsPrecompile.spec.ts]: Done

File: packages/server/tests/acceptance/htsPrecompile.spec.ts (hunk 1 above)
Comment (ar-conmit) [reply packages/server/tests/acceptance/htsPrecompile.spec.ts]: Done

**Review by Nana-EC — APPROVED** [review]
LGTM

**Pull request merged**